data/examples/rasa/demo-rasa.json

```diff
@@ -1,299 +1,197 @@
 {
   "rasa_nlu_data": {
-    "entity_examples": [
+    "common_examples": [
       {
-        "text": "hey", 
-        "intent": "greet", 
+        "text": "hey",
+        "intent": "greet",
         "entities": []
-      }, 
+      },
       {
-        "text": "howdy", 
-        "intent": "greet", 
+        "text": "howdy",
+        "intent": "greet",
         "entities": []
-      }, 
+      },
       {
-        "text": "hey there", 
-        "intent": "greet", 
+        "text": "hey there",
+        "intent": "greet",
         "entities": []
-      }, 
+      },
       {
-        "text": "hello", 
-        "intent": "greet", 
+        "text": "hello",
+        "intent": "greet",
         "entities": []
-      }, 
+      },
       {
-        "text": "hi", 
-        "intent": "greet", 
+        "text": "hi",
+        "intent": "greet",
         "entities": []
-      }, 
+      },
       {
-        "text": "i'm looking for a place to eat", 
-        "intent": "restaurant_search", 
+        "text": "i'm looking for a place to eat",
+        "intent": "restaurant_search",
         "entities": []
-      }, 
+      },
       {
-        "text": "i'm looking for a place in the north of town", 
-        "intent": "restaurant_search", 
+        "text": "i'm looking for a place in the north of town",
+        "intent": "restaurant_search",
         "entities": [
           {
             "start": 31,
-            "end": 36, 
-            "value": "north", 
+            "end": 36,
+            "value": "north",
             "entity": "location"
           }
         ]
-      }, 
-      {
-        "text": "show me chinese restaurants", 
-        "intent": "restaurant_search", 
-        "entities": [
-          {
-            "start": 8, 
-            "end": 15, 
-            "value": "chinese", 
-            "entity": "cuisine"
-          }
-        ]
       },
       {
-        "text": "show me chines restaurants", 
```

Comment:
I think this "interesting" spelling was on purpose to test the synonyms (as the value is different from the found text). I'd either undo that change or create a better synonym example in this test data.